apps/widclkbttm/widclkbttm.wid.js

```diff
@@ -1,6 +1,11 @@
 WIDGETS["wdclkbttm"]={area:"br",width:Bangle.CLOCK?0:60,draw:function() {
+  console.log("clock? "+Bangle.CLOCK);
```

Comment:
Was this intended to be committed?

Reply:
no idea, the condition was no written by me but just copied  from other widget clock (apps\widclk) code. It seems useless, and specially when there a return in case of CLOCK.
The testing consolelog should not be there.. :|